sprint/start_clean.sh

```diff
@@ -4,19 +4,15 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-#Clear terminal screen so the about text can be read
-clear
-
 RED='\033[31m'
 GREEN='\033[32m'
 YELLOW='\033[33m'
 RESET='\033[0m'
 
 # Check Docker is running
-SERVICE='docker'
-if ps ax | grep -v grep | grep -v /Library/PrivilegedHelperTools/com.docker.vmnetd | grep $SERVICE > /dev/null
+if docker run --rm -t busybox:latest ls >/dev/null
 then
-    printf "${GREEN}$SERVICE service running, continuing.\n${RESET}"
+    printf "docker service running, continuing."
```

Comment:
Not related to this line, but we forgot to remove the IDE line in the comments for #71.

```
 # ${GREEN}IDE:       ${YELLOW}http://sprint-[ts].ddev.local:8000/${GREEN}
 #            ${YELLOW}(U:username  P:password)${GREEN}
 #
```

Reply:
Removed, thanks.